### ISSUE TITLE
Getting unwanted hundreds of point acquisitions on iOS

### ISSUE THREAD

**Issue by alexlopespereira** [rekab-app/background_locator#55]
I have an app which is working fine on Android but it is calling the callback functions hundreds of times at each second.

How could I prevent such a behaviour?
Shoud I update the UI in order to dismiss the trigger event?

I'm using this code for setup:
```
BackgroundLocator.registerLocationUpdate(
      LocationBackground.callback,
      androidNotificationCallback: LocationBackground.notificationCallback,
      settings: LocationSettings(
        notificationTitle: "App",
        notificationMsg: "Tracking location",
        notificationIcon: 'ic_background',
        wakeLockTime: 20,
        autoStop: false,
        interval: 60 * 15,
      ),
    );
```

**Comment by alexlopespereira** [comment]
I thought I made some mistake implementing the setup for iOS, but it doesn't seems to be the case. Because I've download the example project from here https://github.com/rekab-app/background_locator/tree/master/example and the same behavior happens on iOS (many many calls to the callback).

How could we prevent this from happening?

**Comment by mehdok** [comment]
Are you testing on _emulator_ or _real device_?
Please give me all the information about testing environment, Include _Xcode version_, _iOS version_, _real device name_ or _simulator_, etc.

Also there is `GpsiesTrack.gpx` in the example project, Are you sure that you are not using it as mock location?

**Comment by alexlopespereira** [comment]
Hi @mehdok I'm testing on a real device, iPhone X, iOS13, Xcode 11.4.1

I'm quite sure I'm not using the GpsiesTrack.gpx file, because these points in the image are the points where I was at the time I ran the app.

**Comment by mehdok** [comment]
This behavior from a real iOS device is really strange. I tried to replicate this situation with iPhone7 and iPhone XS Max, But the they act very normal.

**Comment by alexlopespereira** [comment]
Did you manage to setup triggers to some minutes? I created my environment
again and updated my flutter to 1.17 and it stopped triggering hundreds of
times. But it does not take into account the interval setting. The period
between the triggers keeps been 1 second no matter what.

**Comment by wemersonrv** [comment]
@alexlopespereira did this problem still happens?

Here i have the same issue... So get the example project and run and receive a lot of captures, just like you.

I thing this problem is the cause of of another issue i had a couple of days ago about high battery consumption ... this issue: #51 

Now all makes sense. Over-capture, of course, is pushing the battery a lot. Now it's only a matter of time before Apple screams!  :sob: :joy :innocent

**Comment by mehdok** [comment]
@alexlopespereira @wemersonrv Please test the new version to see if the problem is gone or not.

**Comment by mehdok** [comment]
This should be fixed by #87 ;
